client/src/store/configureStore.dev.js

```diff
@@ -26,4 +26,4 @@ const configureStore = preloadedState => {
   return store;
 };
 
-export default configureStore;
+export default configureStore();
```

Comment:
Why exporting default is calling function instead to invoking. Variable of this module will not hide in closure may produce bugs.